src/core/providers/qgsabstractdatabaseproviderconnection.h

```diff
@@ -513,6 +513,7 @@ class CORE_EXPORT QgsAbstractDatabaseProviderConnection : public QgsAbstractProv
       AddRelationship = 1 << 28,                      //!< Can add new relationships to the database via addRelationship() \since QGIS 3.30
       UpdateRelationship = 1 << 29,                   //!< Can update existing relationships in the database via updateRelationship() \since QGIS 3.30
       DeleteRelationship = 1 << 30,                   //!< Can delete existing relationships from the database via deleteRelationship() \since QGIS 3.30
+      MoveTableToSchema = 1 << 31,                    //!< Can move table to another schema via moveTableToAnotherSchema() \since QGIS 3.44
```

Comment:
@nyalldawson 
I see you've added `Qgis::DatabaseProviderConnectionCapability2` as this enum looks exhausted (`1 << 31` is undefined for `int`), however `1` (`1 << 0`) is never used as the first value used is `1 << 1`!
Would it be OK to decrement all shift values by `1`, or should we assign `MoveTableToSchema` to the unused value `1`?

I'm also curious, could we also utilize the 32nd bit in the future by directly assigning the int value `-2147483648` instead of `1 << 31`?
Would it also work if we used `uint` instead? Reading the docs it seems QT should be happy, but I don't know about SIP?

Reply:
> Would it be OK to decrement all shift values by 1, or should we assign MoveTableToSchema to the unused value 1?

I'm reticent to do that -- there's a moderate chance that someone is using hardcoded ints for these values in some context, so I'd only modify the raw values in a "last resort" situation.

> Would it also work if we used uint instead? Reading the docs it seems QT should be happy, but I don't know about SIP?

It's worth a shot. I can't predict how sip will respond to that :rofl: